pkgs/applications/networking/mailreaders/mailpile/default.nix

```diff
@@ -1,21 +1,22 @@
-{ stdenv, fetchgit, python2Packages, gnupg1orig, makeWrapper, openssl }:
+{ stdenv, fetchgit, python2Packages, gnupg1orig, openssl }:
 
 python2Packages.buildPythonApplication rec {
   name = "mailpile-${version}";
-  version = "0.4.1";
+  version = "0.5.2";
 
   src = fetchgit {
-    url = "git://github.com/pagekite/Mailpile";
+    url = "https://github.com/mailpile/Mailpile";
     rev = "refs/tags/${version}";
-    sha256 = "118b5zwfwmzj38p0mkj3r1s09jxg8x38y0a42b21imzpmli5vpb5";
+    sha256 = "1d2b776x9134sv67pylfkvf1dd4vs5pvgrngpmshrsjgsib13dx5";
   };
 
   patchPhase = ''
```

Comment:
I think its better to have `postPatch` here. That way one can still pass in patches.